extensions/sql-migration/src/constants/strings.ts

```diff
@@ -180,32 +179,32 @@ export const SELECT_BLOB_CONTAINER = localize('sql.migration.blob.container.sele
 export const IR_PAGE_NOTE = localize('sql.migration.ir.page.note', "Note: DMS will run in your Azure subscription in the chosen resource group and does not incur any cost for running it.");
 export const SELECT_A_SQL_MIGRATION_SERVICE = localize('sql.migration.select.a.migration.service', "Select Azure Database Migration Service");
 export const DEFAULT_SETUP_BUTTON = localize('sql.migration.default.setup.button', "Setup with defaults: Add DMS with one click express setup using default options.");
 export const CUSTOM_SETUP_BUTTON = localize('sql.migration.custom.setup.button', "Custom setup: Add DMS after customizing most options.");
-export const SQL_MIGRATION_SERVICE_NOT_FOUND_ERROR = localize('sql.migration.ir.page.sql.migration.service.not.found', "No DMS found. Please create a new one");
+export const SQL_MIGRATION_SERVICE_NOT_FOUND_ERROR = localize('sql.migration.ir.page.sql.migration.service.not.found', "No DMS found. Create a new one");
 export const CREATE_NEW = localize('sql.migration.create.new', "Create new");
-export const INVALID_SERVICE_ERROR = localize('sql.migration.invalid.migration.service.error', "Please select a valid DMS");
-export const SERVICE_OFFLINE_ERROR = localize('sql.migration.invalid.migration.service.offline.error', "Please select a DMS that is connected to a node");
-export const AUTHENTICATION_KEYS = localize('sql.migration.authentication.types', "Authentication Keys");
+export const INVALID_SERVICE_ERROR = localize('sql.migration.invalid.migration.service.error', "Select a valid Data Migration Service.");
+export const SERVICE_OFFLINE_ERROR = localize('sql.migration.invalid.migration.service.offline.error', "Select a Data Migration Service that is connected to a node.");
+export const AUTHENTICATION_KEYS = localize('sql.migration.authentication.types', "Authentication keys");
 export function SQL_MIGRATION_SERVICE_DETAILS_HEADER(sqlMigrationServiceName: string) {
 	return localize('sql.migration.service.header', "Azure Database Migration Service \"{0}\" details:`", sqlMigrationServiceName);
 }
-export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Please note that any existing Azure Database Migration Service (DMS) in Azure portal will not show up in Azure Data Studio. DMS created in Azure Data Studio will not be visible in Azure portal yet.");
+export const DMS_PORTAL_INFO = localize('sql.migration.dms.portal.info', "Any existing Azure Data Migration Services (DMS) in the Azure portal do not appear in Azure Data Studio. Any DMS created in Azure Data Studio will not be visible in the Azure portal yet.");
 export const DATABASE_MIGRATION_SERVICE_AUTHENTICATION_KEYS = localize('sql.migration.database.migration.service.authentication.keys', "Database migration service authentication keys");
```

Comment:
* "Azure Data Migration Services" should be changed to "Azure Database Migration Service" 
* I don't recommend using "DMS" in any strings even (DMS)

Reply:
removed all instances of DMS